iis-wf/iis-wf-transformers/src/test/resources/eu/dnetlib/iis/wf/transformers/metadataextraction/checksum/preprocessing/sampledataproducer/data/output_document_content_url.json

```diff
@@ -1 +1 @@
-{"id": "12345", "url": "http://someurl1.com", "mimeType": "pdf", "contentChecksum": "12345", "contentSizeKB": 0}
\ No newline at end of file
+{"id": "12345", "url": "http://someurl2.com", "mimeType": "pdf", "contentChecksum": "12345", "contentSizeKB": 0}
```

Comment:
Well, it's hard for me to determine if this is problematic in terms of already existing cache.
But there is not much we can do now about it. This change probably happened due to change of pig or some hadoop component (when we switched from cdh4 to cdh5)